gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.sonarsource.scanner.gradle
-version=4.5-SNAPSHOT
+version=4.4.1-SNAPSHOT
```

Comment:
In the release process, they say there is another file to change to prepare release: [integrationTests/pom.xml](https://github.com/SonarSource/sonar-scanner-gradle/blob/master/integrationTests/pom.xml)